lib/main.test.js

```diff
@@ -0,0 +1,79 @@
+import { exec as execCB } from 'child_process';
+import { promisify } from 'util';
+import packageJson from '../package.json';
+
+const exec = promisify(execCB);
+
+const run = (argv) =>
+  exec(
+    [process.execPath, 'build/test-utils/cli-test-entry.js', argv].join(' '),
+  );
+
+describe('cli configuration', () => {
+  beforeAll(() => exec(packageJson.scripts.prepare));
+  it('should show help', async () => {
+    expect((await run('help')).stdout).toMatchInlineSnapshot(`
+      "Usage: resume [command] [options]
+
+      Options:
+        -V, --version                       output the version number
+        -F, --force                         Used by \`publish\` and \`export\` - bypasses
+                                            schema testing.
+        -t, --theme <theme name>            Specify theme used by \`export\` and
+                                            \`serve\` or specify a path starting with .
+                                            (use . for current directory or
+                                            ../some/other/dir) (default:
+                                            \\"jsonresume-theme-even\\")
+        -f, --format <file type extension>  Used by \`export\`.
+        -r, --resume <resume filename>      path to the resume in json format
+                                            (default:
+                                            \\"/Users/thomas.hallock/projects/resume-cli/resume.json\\")
```

Comment:
Won't this line fail in other environments? We probably need to update our `test` job to run on forks as well.

Reply:
oops, I didn't catch that. stand by for an update...

Reply:
now using relative path in --help output